tests/unit/services/page-title-test.js

```diff
@@ -0,0 +1,37 @@
+import { settled } from '@ember/test-helpers';
+import PageTitleService from 'ember-page-title/services/page-title'
+import { setupTest } from 'ember-qunit';
+import { module, test } from 'qunit';
+
+module('Unit | Service | page-title', function(hooks) {
+  setupTest(hooks);
+
+  test('`titleUpdated` hook is called with the new title value', async function(assert) {
+    const titleToken = {
+      id: 1,
+      title: 'Title'
+    };
+
+    let currentTitle;
+
+    class ExtendedPageTitleService extends PageTitleService {
+      titleUpdated(newTitle) {
+        currentTitle = newTitle;
+      }
+    }
+
+    this.owner.register('service:page-title', ExtendedPageTitleService);
+
+    const pageTitleListService = this.owner.lookup('service:page-title-list');
+
+    pageTitleListService.push(titleToken);
+    pageTitleListService.scheduleTitleUpdate();
+    await settled();
```

Comment:
Is there another way to await this? Including `@ember/test-helpers` makes the embroider tests fail it seems.